coalib/results/Diff.py

```diff
@@ -32,6 +32,9 @@ def __init__(self, file_list=None,
         self.delete = delete
         self.create = create
 
+        if create is False and file_list is None:
+            raise ValueError('file_list cannot be None.')
```

Comment:
The user is just using `Diff()`. IMO it doesn't make any sense to create a Diff without a file, if it indeed doesn't create a new file, which will require create to be set